src/App.tsx

```diff
@@ -1,26 +1,49 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { useState, useEffect } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+
+import Header from "./components/Header";
+import Home from "./components/Home";
+import Departments from "./components/Departments";
+import GroupList from "./components/GroupList";
+import Group from "./components/Group";
+import Employee from "./components/Employee";
+import Search from "./components/Search";
+import { EmployeeInterface } from "./components/Employee";
+import EmployeesContext from "./utils/employeesContext";
+import fetchEmployees from "./utils/fetchEmployees";
+import "./sass/main.scss";
+
+const App = (): JSX.Element => {
+  const [employees, setEmployees] = useState<EmployeeInterface[] | null>([]);
```

Comment:
Why does this have to be `EmployeeInterface[] | null`? You can handle the case where `fetchEmployees()` returns `null` and prevent the state from being set.